using-valtimo/upload/upload-to-documenten-api-with-metadata.md

```diff
@@ -34,3 +34,41 @@ export class AppModule {
   ```
 
 Afterwards, the component is available under the advanced section in the form builder.
+
+## Configuring the Case 'upload process'
+
+When a document is uploaded to a case, the case will search for the connected 'upload process' to handle the upload. The
+upload process needs to be configured on the case page:
+
+1. Open the 'Admin' menu.
+2. Go to the 'Cases' page.
+3. Select the case that is going to handle uploads to the Documenten API.
```

Comment:
maybe `Select the case definition that you want to configure the upload process to use the Documenten API for.`